src/components/CommentView/CommentView.tsx

```diff
@@ -214,63 +221,75 @@ export const CommentView: FC<CommentViewProps> = ({
-                            {nullable(!reactionsProps.limited, () => (
-                                <ReactionsDropdown view="icon" onClick={onReactionToggle} />
-                            ))}
-                            <Dropdown
-                                items={dropdownItems}
-                                renderTrigger={({ ref, onClick }) => (
-                                    <IconMoreVerticalOutline
-                                        size="xs"
-                                        className={s.DropdownTrigger}
-                                        ref={ref}
-                                        onClick={onClick}
-                                        {...commentDropdown.attr}
-                                    />
-                                )}
-                                renderItem={({ item, cursor, index }) => (
-                                    <MenuItem
-                                        key={item.label}
-                                        ghost
-                                        color={item.color}
-                                        focused={cursor === index}
-                                        icon={item.icon}
-                                        onClick={item.onClick}
-                                    >
-                                        {item.label}
-                                    </MenuItem>
-                                )}
-                            />
+                            <Dropdown isOpen={isOpenActions} onClose={() => setIsOpenActions(false)}>
+                                <DropdownTrigger
+                                    renderTrigger={({ ref }) => (
+                                        <Button
+                                            size="xs"
+                                            view="ghost"
+                                            className={s.DropdownTriggerButton}
+                                            iconLeft={
+                                                <IconMoreVerticalOutline
+                                                    size="xs"
+                                                    className={s.DropdownTrigger}
+                                                    ref={ref}
+                                                    onClick={() => setIsOpenActions(true)}
+                                                    {...commentDropdown.attr}
+                                                />
+                                            }
+                                        />
+                                    )}
+                                />
+                                <DropdownPanel placement="bottom-start">
+                                    <ListView>
+                                        {dropdownItems.map((item) => (
+                                            <ListViewItem
+                                                key={item.label}
```

Comment:
use services DropdownPanel, you don't need ListView, cause DropdownPanel includes inside it